src/components/TodoItem.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import bin from '../img/bin.png';
+import choonsik from '../img/choonsik.png';
+import editImg from '../img/edit.png';
+function TodoItem({ todoList, todoItem, setTodoList }) {
+  const [edit, setEdit] = useState(false);
+  const [newTodo, setNewTodo] = useState('');
+  const deleteTodo = () => {
+    todoList = todoList.filter((todo) => todo.id !== todoItem.id);
+    setTodoList(todoList);
```

Comment:
```suggestion
     setTodoList(todoList.filter((todo) => todo.id !== todoItem.id));
```

이렇게 하셔도 될 것 같습니다...!